src/code/z_fbdemo_circle.c

```diff
@@ -1,19 +1,134 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_fbdemo_circle/TransitionCircle_Start.s")
+Gfx D_801D0D00[] = { 0xE7000000, 0x00000000, 0xEF002CF0, 0x0342524D, 0xFA0000FF,
+                     0x00000001, 0xF9000000, 0x00000001, 0xDF000000, 0x00000000 };
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_fbdemo_circle/TransitionCircle_Init.s")
+// @bug TransitionCircle_Update may need to be casted? (void (*)(void* transition, s32 updateRate))?
```

Comment:
```suggestion
//! @bug: TransitionCircle_Update should take an additional argument `s32 updateRate`
```
Thoughts on phrasing it like this?